src/graphics.rs

```diff
@@ -85,6 +85,65 @@ pub trait Display: DrawTarget<BinaryColor> {
     }
 }
 
+/// Necessary traits for all displays to implement for drawing
+///
+/// Adds support for:
+/// - Drawing (With the help of DrawTarget/Embedded Graphics)
+/// - Rotations
+/// - Clearing
+pub trait OctDisplay: DrawTarget<OctColor> {
+    /// Clears the buffer of the display with the chosen background color
+    fn clear_buffer(&mut self, background_color: Color) {
+        for elem in self.get_mut_buffer().iter_mut() {
+            *elem = background_color.get_byte_value();
+        }
+    }
```

Comment:
could you try if the clear buffer function could be added to the default trait implemenation? 
Since it seems to be the same function anyway

Reply:
This was an oversight, I have updated the method.